Board.js

```diff
@@ -65,7 +67,12 @@ class Board {
 				if (cell.isHit && cell.hasShip) td.classList.add("hit");
 				if (!preventClicking) {
 					// Each cell has its own event listenser that listens for clicks on itself
-					td.addEventListener("click", e => game.clickSpace(cell, isCurrentPlayer)); 
+					let isVertical = false;
+					document.addEventListener('keydown', (event) => {
```

Comment:
Key controls are cool but I think we should include a button too since intuitive UI is part of the grade